Reviewer/src/main/java/com/softserveinc/reviewer/model/Statistic.java

```diff
@@ -0,0 +1,37 @@
+package com.softserveinc.reviewer.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Statistic {
+    private String methodName;
+    private int invocationCount;
+    private double average;
```

Comment:
I'm pretty sure that it would be enough to have Integer precision of nanoseconds (or even milliseconds) of method execution.